microcosm_pubsub/daemon.py

```diff
@@ -23,27 +23,37 @@ def schema_mappings(self):
         """
         pass
 
-    @abstractproperty
     def handler_mappings(self):
         """
         Define the PubSub message media-type to handler mappings.
 
+        This function is an alternative to configuring `graph.sqs_message_handlers` as
```

Comment:
Most of our daemons have stopped using this property; I'm not yet ready to remove it, but I am making it less required.